lib/menu.ts

```diff
@@ -323,7 +355,7 @@ export function menuItemSubmenu(
     modifiers: {preventOverflow: {padding: 10}},
     boundaries: 'viewport',
     controller: ctl,
-    attach: null,
+    attach: '.' + cssMenuWrap.className,
```

Comment:
Why can't this be just `cssMenuWrap`?

Reply:
`cssMenuWrap` is a function that creates divs with `cssMenuWrap.className` style. What we want is to attach the menu to the nearest `cssMenuWrap`-constructed element that contains the trigger.

Reply:
Ah, right. Thanks!